### PR TITLE
[merged] test: Follow on tests for #155.

### PR DESCRIPTION
**Pull request by ashcrow**

### CODE REVIEW

File: test/test_store_handler_manager.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        """
+        Verify the StoreHandlerManager clone method works as expected.
+        """
+        manager = StoreHandlerManager()
+        manager._registry['test'] = BogusModelType
+        clone_result = manager.clone()
+        # The manager and the cloned_result should not be the same
+        self.assertNotEqual(manager, clone_result)
+        # But their content should be
+        self.assertEqual(manager._registry, clone_result._registry)
```
Comment (mbarnes) [inline test/test_store_handler_manager.py]: I suggest also verify `clone_result._handlers` is `{}` since that's critical for subprocesses.

**Comment by mbarnes** [comment]
Man, I wish I could crank these Mock tests out that fast...

One small suggested change.  Go ahead and merge when you have it, don't wait on me.

**Comment by ashcrow** [comment]
Merging as requested by reviewer.

@rh-atomic-bot r+

**Comment by ashcrow** [comment]
Tagging https://github.com/projectatomic/commissaire/issues/126

**Pull request closed without merging**